content/en/continuous-deployment/release-notes/rn-armory-operator/armory-operator-v1-8-1.md

```diff
@@ -0,0 +1,25 @@
+---
+title: v1.8.1 Armory Operator
+toc_hide: true
+version: 01.08.01
+description: Release notes for Armory Operator v1.8.1
+date: 2022-12-19
+---
+
+## 2022-12-19 Release Notes
+
+## Security
+
+Armory scans the codebase as we develop and release software. For information about CVE scans for this release, contact your Armory account representative.
+
+## Known issues
+
+No known issues.
+
+## Highlighted updates
+
+Operator now supports disabling the Admission Controller.
```

Comment:
I'm assuming someone who uses the Armory Operator would know what "disabling the Admission Controller" means

Reply:
They should know about Admission Controllers, yeah